rst/topics/failover.rst

```diff
@@ -301,6 +301,22 @@ Use your favorite GraphQL client (e.g.
 - ``mutation {cluster{failover_params(){}}}``,
 - ``mutation {cluster{failover_promote()}}``.
 
+E.g. how to setup stateful failover:
```

Comment:
```suggestion
Here is an example of how to setup stateful failover:
```
We [don't use abbreviations like i.e. or e.g.](https://www.tarantool.io/en/doc/latest/contributing/docs/style/#don-t-use-i-e-and-e-g)